devops/deployment/initial_activities.rb

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env ruby
+
+require 'aws-idempotency'
+
+$:.unshift File.expand_path('../lib', File.dirname(__FILE__))
+require 'config'
+
+require 'pp'
+
+
+begin
+
+  ElasticSearch.ensure(elasticsearch_domain)
+
+  endpoint = ElasticSearch.ensure_endpoint_available(elasticsearch_domain)
+
+  puts endpoint
+
+  puts "Ok"
+rescue Aws::AutoScaling::Errors::ServiceError => e
```

Comment:
Can all of these `rescue` calls be combined into a single one, since they all contain the same handling code?